src/app/datagrid/hide-show-columns/hide-show.html

```diff
@@ -77,6 +77,11 @@ <h2>Datagrid: Hide-show columns demo</h2>
     </clr-dg-row>
 
     <clr-dg-footer>
+        <clr-dg-column-toggle>
+            <clr-dg-column-toggle-title>Show Columns!!!</clr-dg-column-toggle-title>
+            <clr-dg-column-toggle-button [type]="'selectAll'">Select All!!!</clr-dg-column-toggle-button>
```

Comment:
You probably know this, but when doing something as static as this with the button types, you can just do `type="selectAll"` instead of `[type]="'selectAll'"`. I'm pointing this because it would be more user-friendly in the documentation.

Reply:
Yup, at one point they were variables and I never flipped it to strings. Fixed up.